src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/StatuteMapper.java

```diff
@@ -3,26 +3,14 @@
 import java.util.List;
 
 import org.mapstruct.Mapper;
-import org.mapstruct.Mapping;
 import org.mapstruct.factory.Mappers;
 
-import ca.bc.gov.open.jag.tco.oracledataapi.dto.StatuteDTO;
-import ca.bc.gov.open.jag.tco.oracledataapi.model.Statute;
-
 @Mapper
 public interface StatuteMapper {
-	
+
 	StatuteMapper INSTANCE = Mappers.getMapper(StatuteMapper.class);
-	
-	@Mapping(source = "statId", target = "id")
-	@Mapping(source = "actCd", target = "actCode")
-	@Mapping(source = "statSectionTxt", target = "sectionText")
-	@Mapping(source = "statSubSectionTxt", target = "subsectionText")
-	@Mapping(source = "statParagraphTxt", target = "paragraphText")
-	@Mapping(source = "statSubParagraphTxt", target = "subparagraphText")
-	@Mapping(source = "statCode", target = "code")
-	@Mapping(source = "statShortDescriptionTxt", target = "shortDescriptionText")
-	@Mapping(source = "statDescriptionTxt", target = "descriptionText")
-	StatuteDTO convertStatute(Statute statute);
-	List<StatuteDTO> convertStatutes(List<Statute> statuteList);
+
+	/** Map from ORDS Statute to OracleDataAPI Statute */
+	ca.bc.gov.open.jag.tco.oracledataapi.model.Statute convertStatute(ca.bc.gov.open.jag.tco.oracledataapi.api.model.Statute statute);
+	List<ca.bc.gov.open.jag.tco.oracledataapi.model.Statute> convertStatutes(List<ca.bc.gov.open.jag.tco.oracledataapi.api.model.Statute> statuteList);
```

Comment:
Can we keep the DTO class and the mapping for that since Phil requested in my previous PR for me to use cleaner names so I have changed the property names on the C# side based on that. So I think in order to keep it compatible and consistent with the C# side, we need to keep the DTO mapping.
You can see Phil's comment here: https://github.com/bcgov/jag-traffic-courts-online/pull/747/files#r973274700

Reply:
Done. Moved StatuteDTO to the model directory and renamed to Statute to be consistent with the other model objects.